webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/AgglomerateFileCache.scala

```diff
@@ -176,7 +176,7 @@ class BoundingBoxCache(
       val isTransformed = Array.fill(input.length)(false)
       while (offset <= readerRange._2) {
         val agglomerateIds: Array[Long] =
-          readHDF(reader, offset, spire.math.min(maxReaderRange, readerRange._2 - offset) + 1)
```

Comment:
What was the original purpose of the use of this external function here?

Reply:
The numbers used to not be Long at that spot, compare https://github.com/scalableminds/webknossos/pull/6532